longhaul-test/message-analyzer-deploy.yml

```diff
@@ -0,0 +1,32 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: message-analyzer-app
+  labels:
+    app: message-analyzer
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: message-analyzer
+  template:
+    metadata:
+      labels:
+        app: message-analyzer
+      annotations:
+        dapr.io/enabled: "true"
+        dapr.io/id: "message-analyzer"
+        dapr.io/port: "5000"
+        dapr.io/log-as-json: "true"
+    spec:
+      containers:
+      - name: message-analyzer
+        image: dapriotest/message-analyzer:dev
+        ports:
+        - containerPort: 5000
```

Comment:
Do we want all apps with the same port? It means they cannot run at the same POD or host, right?

Reply:
I dont think it matters, but to keep it uniform I'll change these to 3000 like the others.